llvm/test/CodeGen/AMDGPU/unpack-non-coissue-insts-post-ra-scheduler.mir

```diff
@@ -0,0 +1,1173 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Can you add an end to end IR test for this? This also doesn't look like it comprehensively tests all the opcodes